mobile/src/main/java/org/openhab/habdroid/util/Constants.java

```diff
@@ -33,7 +33,6 @@
     public static final String PREFERENCE_DEBUG_MESSAGES        = "default_openhab_debug_messages";
     public static final String PREFERENCE_CLEAR_DEFAULT_SITEMAP =
             "default_openhab_clear_default_sitemap";
-    public static final String DEFAULT_GCM_SENDER_ID            = "737820980945";
```

Comment:
*yay* I totally missed the update of myopenhab.org, it's cool that we can now remove this specific value to make the app more generic :)